app/features/utils/functions.js

```diff
@@ -22,3 +23,35 @@ export function openExternalLink(link: string) {
 export function isElectronMac() {
     return process.platform === 'darwin';
 }
+
+/**
+ * Returns the Avatar URL to be used.
+ *
+ * @param {string} key - Unique key to generate Avatar URL.
+ * @returns {string}
+ */
+export function getAvatarURL({ email, id }: {
+    email: string,
+    id: string
+}) {
+    let key = email || id;
+    let urlPrefix;
+    let urlSuffix;
+
+    // If the ID looks like an e-mail address, we'll use Gravatar because it
+    // supports e-mail addresses.
+    if (key && key.indexOf('@') > 0) {
+
+        // URL prefix and suffix of gravatar service.
+        urlPrefix = 'https://www.gravatar.com/avatar/';
+        urlSuffix = '?d=wavatar&size=200';
+    } else {
+        key = id;
+
+        // Otherwise, use a default (meeples, of course).
+        urlPrefix = 'https://abotars.jitsi.net/meeple/';
+        urlSuffix = '';
+    }
+
+    return urlPrefix + md5.hex(key.trim().toLowerCase()) + urlSuffix;
```

Comment:
I know you didn't write this, but can you please use a template string here? thanks!

Reply:
I don't understand.
We are just concatenating variables. Not one of them is a string literal (we don't even have space in between) 
Do you want it to be like `${urlPrefix}${md5.hex(key.trim().toLowerCase())}${urlSuffix}` ?

Reply:
Yep :-)

Reply:
But why?
The template literals were introduced to make string concatenation code easier to read.
If I know correctly, that is only useful when concatenation has string literals and variables. Since we have no strings to concat just variables, this looks better.